Source/Targeting.swift

```diff
@@ -133,6 +133,17 @@ import CoreLocation
         }
     }
     
+    public var deviceAccessConsent: Bool? {
```

Comment:
Am I correct, that a publisher has to handle `deviceAccessConsent` explicitly to get UUID  ?

Reply:
yes the publisher has to explicitly set this purposeConsent to send the IDFA in the request